tests/functional_tests/multisig.py

```diff
@@ -54,30 +56,72 @@ def run_test(self):
+        self.import_multisig_info([0, 1], 7) # seven outputs b/c we're dest plus change
+        self.check_transaction(txid)
+
         self.create_multisig_wallets(2, 3, '44G2TQNfsiURKkvxp7gbgaJY8WynZvANnhmyMAwv6WeEbAvyAWMfKXRhh3uBXT2UAKhAsUJ7Fg5zjjF2U1iGciFk5duN94i')
         self.import_multisig_info([0, 2], 5)
         txid = self.transfer([0, 2])
         self.import_multisig_info([0, 1, 2], 6)
         self.check_transaction(txid)
 
+        self.remake_some_multisig_wallets_by_multsig_seed(2)
+        self.import_multisig_info([0, 2], 6) # six outputs, same as before
+        txid = self.transfer([0, 2])
+        self.import_multisig_info([0, 1, 2], 7) # seven outputs b/c we're dest plus change
+        self.check_transaction(txid)
+
         self.create_multisig_wallets(3, 3, '41mro238grj56GnrWkakAKTkBy2yDcXYsUZ2iXCM9pe5Ueajd2RRc6Fhh3uBXT2UAKhAsUJ7Fg5zjjF2U1iGciFk5ief4ZP')
         self.import_multisig_info([2, 0, 1], 5)
         txid = self.transfer([2, 1, 0])
         self.import_multisig_info([0, 2, 1], 6)
         self.check_transaction(txid)
 
+        self.remake_some_multisig_wallets_by_multsig_seed(3)
+        self.import_multisig_info([2, 0, 1], 6) # six outputs, same as before
+        txid = self.transfer([2, 1, 0])
+        self.import_multisig_info([0, 2, 1], 7) # seven outputs b/c we're dest plus change
+        self.check_transaction(txid)
+
         self.create_multisig_wallets(3, 4, '44vZSprQKJQRFe6t1VHgU4ESvq2dv7TjBLVGE7QscKxMdFSiyyPCEV64NnKUQssFPyWxc2meyt7j63F2S2qtCTRL6dakeff')
         self.import_multisig_info([0, 2, 3], 5)
         txid = self.transfer([0, 2, 3])
         self.import_multisig_info([0, 1, 2, 3], 6)
         self.check_transaction(txid)
 
+        self.remake_some_multisig_wallets_by_multsig_seed(3)
+        self.import_multisig_info([0, 2, 3], 6) # six outputs, same as before
+        txid = self.transfer([0, 2, 3])
+        self.import_multisig_info([0, 1, 2, 3], 7) # seven outputs b/c we're dest plus change
+        self.check_transaction(txid)
+
         self.create_multisig_wallets(2, 4, '47puypSwsV1gvUDratmX4y58fSwikXVehEiBhVLxJA1gRCxHyrRgTDr4NnKUQssFPyWxc2meyt7j63F2S2qtCTRL6aRPj5U')
         self.import_multisig_info([1, 2], 5)
         txid = self.transfer([1, 2])
         self.import_multisig_info([0, 1, 2, 3], 6)
         self.check_transaction(txid)
 
+        self.remake_some_multisig_wallets_by_multsig_seed(2)
+        self.import_multisig_info([0, 1, 2, 3], 6) # six outputs, same as before
+        txid = self.transfer([2, 3])
+        self.import_multisig_info([0, 1, 2, 3], 7) # seven outputs b/c we're dest plus change
+        self.check_transaction(txid)
+
+        self.create_multisig_wallets(1, 2, '4A8RnBQixry4VXkqeWhmg8L7vWJVDJj4FN9PV4E7Mgad5ZZ6LKQdn8dYJP2RePX6HMaSkbvTbrWUFhDNcNcHgtNmQ4S8RSB')
+        self.import_multisig_info([0, 1], 5)
+        txid = self.transfer([0])
+        self.import_multisig_info([0, 1], 6)
+        self.check_transaction(txid)
```

Comment:
I added a test case for a 1/2 multisig wallet since none of the cases yet have 1 as the threshold